website/docs/Pipelines/SlideSeq_Pipeline/README.md

```diff
@@ -0,0 +1,234 @@
+
+<!--- #### Sample inputs for analyses in a Terra Workspace --->
+
+## SlideSeq tasks and tools
+
+The [SlideSeq workflow](https://github.com/broadinstitute/warp/blob/master/pipelines/skylab/slide_seq/SlideSeq.wdl) imports individual "tasks," also written in WDL script, from the WARP [tasks folder](https://github.com/broadinstitute/warp/tree/master/tasks/skylab). 
+
+Overall, the SlideSeq workflow:
+1. Calculates prealignment metrics.
+1. Filters, trims and splits reads into < 30 GB FASTQs with sctools.
+1. Corrects bead barcodes, aligns reads, and counts genes with STARsolo.
+1. Calculates metrics.
+1. Merges the Star outputs into NPY and NPZ arrays.
+1. Merges gene counts, metrics, and bead locations into a Loom-formatted matrix.
+
+The tools each SlideSeq task employs are detailed in the table below. 
+
+To see specific tool parameters, select the task WDL link in the table; then find the task and view the `command {}` section of the task in the WDL script. To view or use the exact tool software, see the task's Docker image which is specified in the task WDL `# runtime values` section as `String docker =`.
+
+| Task name and WDL link | Tool | Software | Description | 
+| --- | --- | --- | --- | 
+| [Metrics.FastqMetricsSlidSeq (alias = FastqMetrics)](https://github.com/broadinstitute/warp/blob/master/tasks/skylab/Metrics.wdl) | fastq_metrics | [sctools](https://github.com/HumanCellAtlas/sctools) | Calculates prealignment metrics from the input FASTQ files. Outputs metrics files including the bead barcode distribution, UMI distribution, number of reads per cell and number of UMIs per cell. |
+| [FastqProcessing.FastqProcessingSlidSeq (alias = SplitFastq)](https://github.com/broadinstitute/warp/blob/master/tasks/skylab/FastqProcessing.wdl) | fastq_slideseq | [sctools](https://github.com/HumanCellAtlas/sctools) | Filters reads, trims bead barcodes in read 1 to remove spacer sequences, and partitions the input FASTQ files by bead barcode to create an array of FASTQ files that are each ~ 30 GB. The function keeps all reads belonging to the same bead barcode in the same partitioned FASTQ file.| 
+| [StarAlign.STARsoloFastqSlideSeq (alias = STARsoloFastqSlideSeq)](https://github.com/broadinstitute/warp/blob/master/tasks/skylab/StarAlign.wdl) | STAR | [Star](https://github.com/alexdobin/STAR) | For each of the partitioned FASTQ files, performs bead barcode correction, poly(A) tail trimming from adapters, alignment, gene annotation, and gene counting. Produces a count matrix and BAM file for each partition.|
```

Comment:
Here we're saying bead barcode correction again, so we might need to update that to make it clear it's filtering, not correcting?

Reply:
Correcting is actually happening in STARsolo. I think there was just some confusion about the wording above the table.